dev/Breadcrumb/Breadcrumb.cpp

```diff
@@ -0,0 +1,561 @@
+    m_breadcrumbElementFactory = winrt::make_self<BreadcrumbElementFactory>();
+}
+
+void Breadcrumb::OnApplyTemplate()
+{
+    winrt::IControlProtected controlProtected{ *this };
+
+    // TODO: Implement
+    m_breadcrumbItemRepeater.set(GetTemplateChildT<winrt::ItemsRepeater>(L"BreadcrumbItemRepeater", controlProtected));
+
+    if (auto const& thisAsIUIElement7 = this->try_as<winrt::IUIElement7>())
+    {
+        thisAsIUIElement7.PreviewKeyDown({ this, &Breadcrumb::OnChildPreviewKeyDown });
+    }
+
+    AccessKeyInvoked({ this, &Breadcrumb::OnAccessKeyInvoked });
+    GettingFocus({ this, &Breadcrumb::OnGettingFocus });
+
+    if (const auto& breadcrumbItemsRepeater = m_breadcrumbItemRepeater.get())
+    {
+        breadcrumbItemsRepeater.ItemsSource(winrt::make<Vector<IInspectable>>());
+        breadcrumbItemsRepeater.ItemTemplate(*m_breadcrumbElementFactory);
+        
+        m_itemRepeaterElementPreparedRevoker = breadcrumbItemsRepeater.ElementPrepared(winrt::auto_revoke, { this, &Breadcrumb::OnElementPreparedEvent });
+        m_itemRepeaterElementIndexChangedRevoker = breadcrumbItemsRepeater.ElementIndexChanged(winrt::auto_revoke, { this, &Breadcrumb::OnElementIndexChangedEvent });
+        m_itemRepeaterElementClearingRevoker = breadcrumbItemsRepeater.ElementClearing(winrt::auto_revoke, { this, &Breadcrumb::OnElementClearingEvent });
+
+        m_breadcrumbItemRepeaterLoadedRevoker = breadcrumbItemsRepeater.Loaded(winrt::auto_revoke, { this, &Breadcrumb::OnBreadcrumbItemRepeaterLoaded });
+    }
+
+    UpdateItemsSource();
+}
+
+void Breadcrumb::OnPropertyChanged(const winrt::DependencyPropertyChangedEventArgs& args)
+{
+    const winrt::IDependencyProperty& property = args.Property();
+
+    // TODO: Implement
+    if (property == s_ItemsSourceProperty)
+    {
+        UpdateItemsSource();
+    }
+    else if (property == s_ItemTemplateProperty)
+    {
+        UpdateItemTemplate();
+    }
+    else if (property == s_DropdownItemTemplateProperty)
+    {
+        UpdateDropdownItemTemplate();
+    }
+}
+
+void Breadcrumb::OnBreadcrumbItemRepeaterLoaded(const winrt::IInspectable&, const winrt::RoutedEventArgs&)
+{
+    if (const auto& breadcrumbItemRepeater = m_breadcrumbItemRepeater.get())
+    {
+        OnRepeaterCollectionChanged(nullptr, nullptr);
+    }
+}
+
+void Breadcrumb::UpdateItemTemplate()
+{
+    const winrt::IInspectable& newItemTemplate = ItemTemplate();
+    m_breadcrumbElementFactory->UserElementFactory(newItemTemplate);
+}
+
+void Breadcrumb::UpdateDropdownItemTemplate()
+{
+    const winrt::IInspectable& newItemTemplate = DropdownItemTemplate();
+
+    // Copy the item template to the ellipsis button too
+    if (const auto& ellipsisBreadcrumbItem = m_ellipsisBreadcrumbItem.get())
+    {
+        if (const auto& itemImpl = winrt::get_self<BreadcrumbItem>(ellipsisBreadcrumbItem))
+        {
+            itemImpl->SetFlyoutDataTemplate(newItemTemplate);
+        }
+    }
+}
+
+void Breadcrumb::UpdateItemsSource()
```

Comment:
UpdateItemsRepeaterItemsSource()

(you're not updating Breadcrumb.ItemsSource)